contrib/verify-commits/README.md

```diff
@@ -33,6 +33,17 @@ Configuration files
 * `trusted-keys`: This file should contain a \n-delimited list of all PGP fingerprints of authorized commit signers (primary, not subkeys).
 * `allow-revsig-commits`: This file should contain a \n-delimited list of git commit hashes. See next section for more info.
 
+Import trusted keys
+-------------------
+In order to check the commit signatures you must add the trusted PGP keys to your machine. This can be done in Linux by running
+    while read -r LINE; do gpg --recv-keys $LINE; done < contrib/verify-commits/trusted-keys
+
+Note for smartcard users
+----------------------
+If you use a smartcard such as a YubiKey for GPG signing that requires a
+physical button press, you will need to set gpgSign = false in your git config,
```

Comment:
"gpgSign = false" better formatted as `gpgSign = false`? :-)

Reply:
Yes, though I've completed removed this section now because @Sjors's suggested change to the git merge command in the python script now dynamically disables gpg signing